integration/blocked_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"github.com/go-check/check"
+)
+
+const blockedRegistriesConf="./fixtures/blocked-registries.conf"
+
+func (s *SkopeoSuite) TestCopyBlockedSource(c *check.C) {
+	assertSkopeoFails(c, "", "--registries-conf="+blockedRegistriesConf, "copy",
```

Comment:
I would expect a non-empty regex here, to distinguish between uses of a blocked registry and accesses to nonexistent `registry-unblocked.com` failing.

Or, maybe more realistic (although slower), actually use a real registry with accessible images.

(Absolutely non-blocking: Can’t this use `"--registries-conf", blockedRegistriesConf` instead of the concatenation?)